Environment/AeroAllergenObserved/doc/spec.md

```diff
@@ -0,0 +1,172 @@
+
++ `id` : Unique identifier. 
+
++ `type` : Entity type. It must be equal to `AeroAllergenObserved`.
+
++ `dateModified` : Last update timestamp of this entity.
+    + Attribute type: [DateTime](https://schema.org/DateTime)
+    + Optional
+
++ `dateCreated` : Entity's creation timestamp.
+    + Attribute type: [DateTime](https://schema.org/DateTime)
+    + Optional    
+
++ `location` : Location of the aero allergens observation represented by a GeoJSON geometry. 
+    + Attribute type: `geo:json`.
+    + Normative References: [https://tools.ietf.org/html/rfc7946](https://tools.ietf.org/html/rfc7946)
+    + Mandatory if `address` is not defined. 
+    
++ `address` : Civic address of the aero allergens observation. Sometimes it corresponds to the aero allergens station address.
+    + Normative References: [https://schema.org/address](https://schema.org/address)
+    + Mandatory if `location` is not present. 
+    
++ `dateObserved` : The date and time of this observation in ISO8601 UTCformat. It can be represented by an specific time instant or by an ISO8601 interval. 
+    + Attribute type: [DateTime](https://schema.org/DateTime) or an ISO8601 interval represented as [Text](https://schema.org/Text). 
+    + Mandatory
+    
++ `source` : A sequence of characters giving the source of the entity data.
+    + Attribute type: [Text](https://schema.org/Text) or [URL](https://schema.org/URL)
+    + Optional
+    
++ `allergenRisk` : Overall allergen risk corresponding to the aero allergens observed.
+    + Attribute type: [Text](https://schema.org/Text)
+    + Example values defined by the [European Aeroallergen Network](https://www.ean-net.org/en/):
+  (`none`, `low`, `moderate`, `high`, `very high`).
+  As this can be different between countries, regulations or implementations, the set of allowed values will depend on the reference specification used.
+  It is recommended that implementations use the same naming conventions as exemplified above (lower case starting words, camel case when compound terms are used)
+    + Attribute metadata:
+        + `referenceSpecification` : Specification that must be taken as reference when interpreting the supplied qualitative value. 
+            + Type: [Text](https://schema.org/Text) or [URL](https://schema.org/URL)
+            + Mandatory
+    + Optional
+  
++ `refDevice` : A reference to the device(s) which captured this observation.
+    + Attribute type: Reference to an entity of type `Device`
+    + Optional
+
+### Representing aero allergens concentration
+
+In order to enable a proper management of the the different aero allergens concentration,
+*for each* aero allergens we have to use an attribute that *MUST* refers exactly
+to the conventional name of the allergen (usually the latin name of
+the associated plant) concatenated with the string `Concentration`,
```

Comment:
as I said before we don't need to concatenate the word 'concentration', just use, as convention, the name of the allergen as the key